core/src/main/java/com/onelogin/saml2/logout/LogoutRequest.java

```diff
@@ -456,6 +456,17 @@ public Boolean isValid() throws Exception {
 				if (signAlg == null || signAlg.isEmpty()) {
 					signAlg = Constants.RSA_SHA1;
 				}
+
+				if (signAlg.equals(Constants.RSA_SHA1)) {
+					Boolean rejectDeprecatedAlg = settings.getRejectDeprecatedAlg();
+					if (rejectDeprecatedAlg) {
+						LOGGER.error("A deprecated algorithm (RSA_SHA1) found in the Signature element, rejecting it");
```

Comment:
I wonder if it can be confusing that there are 2 kinds of rejections, the other being:
```java
			if (!isAlgorithmWhitelisted(sigMethodAlg)){
				throw new Exception(sigMethodAlg + " is not a valid supported algorithm");
			}
```

Reply:
Supported by the toolkit vs Acceptable